TrackletAlgorithm/TrackletEngineUnit.h

```diff
@@ -31,22 +31,52 @@ class TrackletEngineUnit {
 
 #pragma HLS ARRAY_PARTITION variable=stubptinnerlutnew_ complete dim=1
 #pragma HLS ARRAY_PARTITION variable=stubptouterlutnew_ complete dim=1
+/////  Grabs the appropriate lut based on seed and iTC (need to be included in download.sh)
 
-
-    idle_ = true;
-
-    ap_uint<1> stubptinnertmp[kNBitsPTLut] =
+    if (Seed==TF::L1L2&&iTC==3){
+      ap_uint<1> stubptinnertmp[256]=
```

Comment:
@jasonfan393 Please remove the 256 specifying the size of the array. It's a magic number and isn't necessary when the array is being initialized at the same time.

This should also be an array of type `const ap_uint<1>`, and these comments apply to the other "tmp" arrays defined below.